fdbclient/NativeAPI.actor.cpp

```diff
@@ -7733,49 +7732,50 @@ ACTOR Future<std::vector<CheckpointMetaData>> getCheckpointMetaData(Database cx,
                                                                     CheckpointFormat format,
                                                                     double timeout) {
 	state Span span("NAPI:GetCheckpoint"_loc);
+	state int index = 0;
+	state std::vector<Future<CheckpointMetaData>> futures;
```

Comment:
I moved these two because they are used outside of the below loop and that was confusing my IDE.